File: app/src/main/java/com/starostinvlad/rxeducation/GsonModels/Images.java
package com.starostinvlad.rxeducation.GsonModels;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import androidx.room.Ignore;

public class Images implements Serializable {

    @SerializedName("small")
    @Expose
    @Ignore
    private String small;
    @SerializedName("medium")
    @Expose
    private String medium;
    @SerializedName("large")
    @Expose
    @Ignore
    private String large;

    public String getSmall() {
        return small;
    }

    public void setSmall(String small) {
        this.small = small;
    }

    public String getMedium() {
        return medium;
    }

    public void setMedium(String medium) {
        this.medium = medium;
    }

    public String getLarge() {
        return large;
    }

    public void setLarge(String large) {
        this.large = large;
    }

}

File: app/src/main/java/com/starostinvlad/fan/GsonModels/Images.kt
package com.starostinvlad.fan.GsonModels

import androidx.room.Ignore
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import kotlin.Throws

class Images : Serializable {
    @SerializedName("small")
    @Expose
    @Ignore
    var small: String? = null

    @SerializedName("medium")
    @Expose
    var medium: String? = null

    @SerializedName("large")
    @Expose
    @Ignore
    var large: String? = null
}